packages/bpk-component-infinite-scroll/src/withInfiniteScroll-test.js

```diff
@@ -204,18 +204,6 @@ describe('withInfiniteScroll', () => {
     expect(currentOptions).toEqual({ threshold: 0.99 });
   });
 
-  it('should create the IntersectionObserver with a 0.99 threshold if the prop is set with an invalid value', () => {
```

Comment:
Not sure about this: on one side it's weird to add a test to check the behaviour when a prop is invalid, but on the other it seems safer 😂